### DISCUSSION TITLE
Wrong callout for `Border Width` section

### DISCUSSION THREAD

**Discussion by danielberndt** [chakra-ui/panda#2533] [General]
### Subject

Tokens

### Description

![image](https://github.com/chakra-ui/panda/assets/856386/85e26039-409e-44a8-9bdf-5c8950065e7d)

The [Border Width's](https://panda-css.com/docs/theming/tokens#border-widths) callout is a duplicate of the `Radii`'s callout

**Comment by anubra266** [comment]
@danielberndt could you open a PR that removes the callout.

**Reply by danielberndt** [reply]
Sure, just created https://github.com/chakra-ui/panda/pull/2534
